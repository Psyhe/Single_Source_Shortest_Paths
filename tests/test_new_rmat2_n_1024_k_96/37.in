1024 407 417
407 294 241
407 395 81
407 468 13
407 390 165
407 302 45
407 932 201
407 214 176
407 39 2
407 504 172
407 150 228
407 399 197
407 405 245
407 503 84
407 902 186
408 912 73
408 384 161
408 393 223
408 280 110
408 412 119
408 920 109
408 28 230
408 474 125
408 904 21
408 432 155
408 401 242
408 153 221
408 648 255
408 88 160
408 320 158
408 280 26
408 392 67
408 145 126
408 413 117
408 380 116
408 472 127
408 398 55
408 789 50
408 280 3
409 411 94
409 400 230
409 447 20
409 905 192
409 217 156
409 793 27
409 441 103
409 236 71
409 488 118
409 273 5
409 145 196
409 329 36
409 921 45
410 276 238
410 170 225
410 507 82
410 27 6
410 394 187
410 920 44
410 414 203
410 26 66
410 976 169
410 299 153
410 413 108
410 155 74
410 154 91
411 906 215
411 155 166
411 443 64
411 315 13
411 899 225
411 435 84
411 339 42
411 153 171
411 397 104
411 403 235
411 409 94
411 155 148
412 510 212
412 156 9
412 156 172
412 821 190
412 156 123
412 136 95
412 136 97
412 280 204
412 280 96
412 284 215
412 413 119
412 440 102
412 408 119
412 240 251
413 412 119
413 188 7
413 177 39
413 285 115
413 410 108
413 401 120
413 479 160
413 392 39
413 408 117
414 158 169
414 398 224
414 410 203
415 232 76
415 151 26
415 395 251
415 399 6
415 700 65
415 183 12
416 928 205
416 352 142
416 896 220
416 162 108
416 289 21
416 256 92
416 424 204
416 161 51
416 289 60
416 928 102
416 484 63
416 422 206
416 820 238
416 492 143
416 160 46
416 132 246
416 800 186
416 336 67
416 192 237
416 432 238
416 674 134
417 305 161
417 403 185
417 268 217
417 801 143
417 384 75
417 163 0
417 257 119
417 421 227
417 401 245
417 481 255
417 160 61
417 801 176
417 161 130
417 97 242
417 897 215
417 432 62
417 177 1
417 144 191
417 165 63
417 393 164
