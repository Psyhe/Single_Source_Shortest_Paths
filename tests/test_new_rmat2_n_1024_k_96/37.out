51
35
43
72
80
51
60
275
37
52
22
